client/src/Ratings.jsx

```diff
@@ -1,59 +1,65 @@
+/* eslint-disable import/extensions */
 import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import axios from 'axios';
+
 import ReviewAction from './RatingsComponent/ReviewAction.jsx';
 import RatingsSearch from './RatingsComponent/RatingsSearch.jsx';
 import RatingsContent from './RatingsComponent/RatingsContent.jsx';
 import RatingsStarHeader from './RatingsComponent/RatingsStarHeader.jsx';
 import RatingsSummaryReview from './RatingsComponent/RatingsSummaryReview.jsx';
-const { data } = require('./RatingsComponent/RatingsDummyData.js');
-
-let newData = data.slice(0,2);
 
 class Ratings extends Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
     this.state = {
```

Comment:
Why not use React hooks with useState here as well to match more of the team coding goals / what you have in the other files? (honestly curious)

Reply:
This ties into how we are going to pass around the data. Still trying to figure out how to use context and how that relates to my form submission, but will refactor the codebase based on our discussion today.